articles/api-auth/tutorials/use-one-rs-for-apis.md

```diff
@@ -0,0 +1,108 @@
+---
+  description: How to use multiple APIs and represent them as a single Resource Server in Auth0.
+---
+
+# Represent Multiple APIs as a Single Resource Server
+
+If you have multiple APIs, you can treat them as a single Resource Server. Doing this allows you to implement just one authentication flow. You can then control access to the individual APIs by assigning the appropriate scopes.
+
+This article shows you how to use and represent multiple APIs as a single Resource Server in Auth0 using a [sample application you can download](https://github.com/auth0-samples/auth0-api-auth-implicit-sample) if you would like to follow along as you read.
+
+## The Sample Application
+
+In the sample application, we have:
```

Comment:
In the sample application you only have the first bullet, the two API js files. The other entries are things involved in this sample application (terms and concepts). 

I would remove this and add a couple of sentences describing the sample: `The sample contains 1 SPA and 2 APIs: contacts and calendar. We will represent the APIs in Auth0 with a single API, which we will name Organizer Service. Then we will configure the scopes .... and see how you can use the Implicit Grant to access these two APIs from the SPA.`